README.md

```diff
@@ -744,6 +748,13 @@ A new yml file should be created in `marts/` which contains all tests and docume
         ├── staging.yml
 ```
 
+## Performance
```

Comment:
assuming these are just placeholders?

Reply:
Yep! Was waiting to add documentation until we confirmed which strategy we wanted to move forward with

Reply:
(And once we've determined the appropriate value for `chained_views_threshold` so I can give a clear example)